tests/integration/actions/test_generate_schema_change_expectations.py

```diff
@@ -66,29 +104,12 @@ def token_env_var_local():
     return os.environ.get("GX_CLOUD_ACCESS_TOKEN")
 
 
-@pytest.fixture
-def local_mercury_db_datasource(
-    context: CloudDataContext,
-) -> PostgresDatasource:
-    datasource_name = "local_mercury_db"
-    datasource = context.data_sources.get(datasource_name)
-    yield datasource
-
-
-@pytest.fixture
-def local_mercury_db_organizations_table_asset(
-    local_mercury_db_datasource: PostgresDatasource,
-) -> TableAsset:
-    data_asset_name = "local-mercury-db-organizations-table"
-    data_asset = local_mercury_db_datasource.get_asset(name=data_asset_name)
-    yield data_asset
-
-
+@pytest.mark.skip(
+    "This test fails due to a connection error when attempting to call the backend. It is a problem with the test setup, not the code which was demoed, and this test passes locally."
+)
```

Comment:
Test is marked skipped until we can figure out the issues running on the CI infrastructure.

Reply:
Sounds good. Really appreciate the work that you've done with this.

Reply:
Here is the ticket to re-enable: https://greatexpectations.atlassian.net/browse/ZELDA-1175